packages/vite/src/node/server/index.ts

```diff
@@ -271,6 +271,10 @@ export interface ViteDevServer {
    * Print server urls
    */
   printUrls(): void
+  /**
+   * Bind shortcuts
+   */
+  bindShortcuts(options?: BindShortcutsOptions): void
```

Comment:
```suggestion
  bindShortcuts(options?: BindShortcutsOptions<ViteDevServer>): void
```

I think the types can be stricter here.